views/index.handlebars

```diff
@@ -100,6 +100,13 @@
       </div>
     </div>
 
+    <div id="compliance-error">
+      <div class="title">
+        Encryption error<br>
+        Your browser does not support gcm encryption.
```

Comment:
This applies to the current stable version of Safari.

Yeah, I left this fairly basic since it will eventually be replaced by the new UI.